### PR TITLE
DL-7: Create Text Component

### PR DESCRIPTION
**Pull request by A-Hanchar**
https://trello.com/c/2cmuBtmz/7-create-text-component

### CODE REVIEW

**Comment by A-Hanchar** [comment]
[Create Text Component](https://trello.com/c/2cmuBtmz/7-create-text-component)

File: src/atoms/Text/__stories__/Text.stories.tsx [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
 
 export default {
   title: 'Atoms/Text',
   component: Text,
-} as ComponentMeta<typeof Text>
+  args: {
+    as: 'p',
+    children: 'Far far away, behind the word mountains, far from the countries Vokalia and Consonantia.',
+  },
+  tags: ['autodocs'],
```
Comment (sohodoll) [inline src/atoms/Text/__stories__/Text.stories.tsx]: need to remove this

File: src/atoms/Text/__tests__/Text.test.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    )
+
+  it('SHOULD render component', () => {
+    renderTextComponent()
+
+    expect(screen.getByTestId(drafterId)).toBeInTheDocument()
+  })
+
+  describe('check tag name', () => {
+    it('SHOULD render correct tag')
```
Comment (sohodoll) [inline src/atoms/Text/__tests__/Text.test.tsx]: please remove this 'it'

File: src/atoms/Text/__stories__/Text.stories.tsx (hunk 1 above)
Comment (A-Hanchar) [reply src/atoms/Text/__stories__/Text.stories.tsx]: done

File: src/atoms/Text/__tests__/Text.test.tsx (hunk 2 above)
Comment (A-Hanchar) [reply src/atoms/Text/__tests__/Text.test.tsx]: done

**Review by sohodoll — APPROVED** [review]

**Pull request merged**